src/diffusers/pipelines/cogvideo/pipeline_cogvideox.py

```diff
@@ -585,11 +613,6 @@ def __call__(
             `tuple`. When returning a tuple, the first element is a list with the generated images.
         """
 
-        if num_frames > 49:
```

Comment:
should we make num_frames a config of the transformer model then? 
that way we can config 49 for 2b checkpoint and 89 for 5b

Reply:
Let's not put a limit here actually. This was only to provide a cleaner error message instead of the crash message in the transformer when num_frames was > 49. I have some ideas in mind about temporal tiling in transformer to allow longer video generation (but I will keep that for a separate discussion and PR if I find the time, and if the changes are not too clunky. It could be a feature similar to how FreeNoise is for AnimateDiff).